sdk/python/requirements/py3.10-ci-requirements.txt

```diff
@@ -208,22 +204,24 @@ google-auth==1.35.0
     #   google-cloud-storage
 google-auth-httplib2==0.1.0
     # via google-api-python-client
-google-auth-oauthlib==0.5.0
+google-auth-oauthlib==0.5.1
     # via gcsfs
-google-cloud-bigquery==2.34.1
-    # via feast (setup.py)
-google-cloud-bigquery-storage==2.12.0
+google-cloud-bigquery==3.0.1
```

Comment:
We should pin this below 3.0.0. I think the 3+ is causing some issues, cc #2537 or #2532

Reply:
Sure, let me do that right now.